tidy3d/web/api/container.py

```diff
@@ -701,8 +711,19 @@ def pbar_description(task_name: str, status: str) -> str:
             "postprocess",
             "visualize",
             "success",
```

Comment:
@tylerflex why the success is running status?

Reply:
it's just the final stage of the task running. It's a bit different logic from end_statuses. `run_statuses` are the various orders steps of running a task. The `end_statuses` are the possible final stages (when we know to break out of the monitoring).